changelogutils/changelog_test.go

```diff
@@ -4,7 +4,6 @@ import (
 	"bytes"
 	"context"
 	"fmt"
-	"io/ioutil"
```

Comment:
This has been deprecated for ages and the version of golang used here supports the replacements.